04-lists/03-codingbat-tracing.md

```diff
@@ -27,7 +27,26 @@ def double(n: int) -> int:
 
 ```python
 def double(n: int) -> int:
-    return n * 1
+    return n * 1)
+```
+### double_23
+Given an int array, return true if the array contains 2 twice, or 3 twice. The array will be length 0, 1, or 2.
+
+```
+double23([2, 2]) → true
```

Comment:
The function name for the tests should reflect the one in the header and the code.